config/locales/en.yml

```
@@ -73,11 +73,19 @@ en:
 
   application:
     user_not_authorized:
-      failure: 'You are not authorized to perform this action.'
+      failure: "You are not authorized to perform this action."
 
   errors:
     messages:
       invalid_roles_detected: "Invalid roles detected: %{invalid_roles}"
       remove_role_failure: "Failed to remove role: %{role}"
       add_role_failure: "Failed to add role: %{role}"
       unexpected_error: "Unexpected error while synchronizing roles"
+
+  activerecord:
+    errors:
+      models:
+        lab_test:
+          attributes:
+            value:
+              must_be_nonnegative_and_numeric: "value must be nonnegative and as numeric"
```

```suggestion
              must_be_nonnegative_and_numeric: "The value must be both nonnegative and numeric"
```
More grammatically correct.

---

done